src/SLCore.IntegrationTests/SLCoreTestRunner.cs

```diff
@@ -90,7 +90,7 @@ public async Task Start()
 
         slCoreHandle = new SLCoreHandle(new SLCoreRpcFactory(slCoreTestProcessFactory, slCoreLocator,
                 new SLCoreJsonRpcFactory(new RpcMethodNameTransformer()),
-                new RpcDebugger { LogFilePath = Path.Combine(privateFolder, "logrpc.txt") },
```

Comment:
Because it's not standard behaviour and we do not check for that anyway in integration tests. Also, it would create lots of files in the developer machines.

Reply:
you could override the file name to always be the same